json-endpoints/jetpack/class.jetpack-json-api-cron-endpoint.php

```diff
@@ -200,7 +200,7 @@ protected function result() {
 		$lock = $this->lock_cron();
 		$next = wp_schedule_single_event( $args['timestamp'], $hook, $arguments );
 		$this->maybe_unlock_cron( $lock );
-		return array( 'success' => is_null( $next  ) ? true : false );
+		return array( 'success' => false !== $next );
```

Comment:
It would be worthwhile adding a comment noting the pre-5.1 and post-5.1 difference we need to keep in mind here.

```
// Note: Before WP 5.1, the return value was either `false` or `null`.
// With 5.1 and later, the return value is now `false` or `true`.
// We need to account for both.
```

Reply:
👍 	done in 5becda4de931b82fa212c3f83688f0174397cdf8